### PR TITLE
Rework ::pos to allow a user to return another player's position

### LINKED ISSUE #409: Command to get another player's position
Expand the existing `::pos` command to support `::pos [player]`, which displays the position of the specified player.

- Should require moderator rights to use.
- Should display something like "Player [playername] is not currently online." if the specified player is not logged in.

Mentoring from the project maintainers is available for anyone who wishes to work on this issue - please join our Slack team (link in the main project README) and let us know.

### PR DESCRIPTION
**Pull request by KeepBotting**
Closes #409. This is my first time working with Kotlin so please do point out anything that could be improved.

Here's an [image](http://netherworld.pw/files/YzM4Zjk.png) demonstrating functionality.

### CODE REVIEW

**Review by garyttierney — CHANGES_REQUESTED** [review]
Let's add a test case for this new functionality: https://github.com/apollo-rsps/apollo/blob/9923a4c14c1a6d38182b484636caee6f79e9715b/game/plugin/cmd/test/TeleportCommandTests.kt

File: game/plugin/cmd/src/teleport-cmd.plugin.kts [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+            return@then
+          }
+        } else {
+          target = player
+        }
+
+        val (x, y, z) = target.position
+        val region = target.position.regionCoordinates
+
+        player.sendMessage("${target.getUsername().capitalize()} is located at ($x, $y, $z) in region (${region.x}, ${region.y}).")
```
Comment (garyttierney) [inline game/plugin/cmd/src/teleport-cmd.plugin.kts]: Drop the `capitalize()`

Comment (garyttierney) [reply game/plugin/cmd/src/teleport-cmd.plugin.kts]: And use property access syntax: `target.username`

Comment (KeepBotting) [reply game/plugin/cmd/src/teleport-cmd.plugin.kts]: Done!

**Comment by KeepBotting** [comment]
Tests added!

**Pull request merged**